### PR TITLE
Copy of interesting data

### PR DESCRIPTION
**Pull request by datanel**
- fusio data are now stored in another data dir
- added a conf for a logger
- fixed typo (datahub -> tartare)
- moved integration tests to tests/integration

### CODE REVIEW

File: tartare/__init__.py [hunk 1]
```diff
@@ -28,16 +28,32 @@
...
+import logging
+import logging.config
 
+import sys
 from flask import Flask
 
 from tartare.helper import make_celery
 from celery.signals import setup_logging
 
+def configure_logger(app):
```
Comment (kinnou02) [inline tartare/__init__.py]: This should probably be elsewhere, like in helpers.py

File: tartare/default_settings.py [hunk 2]
```diff
@@ -30,3 +30,33 @@
...
+CURRENT_DATA_DIR='./current'
+
+LOGGER = {
+    'version': 1,
+    'disable_existing_loggers': True,
+    'formatters': {
+        'default': {
+            'format': '[%(asctime)s] [%(levelname)5s] [%(process)5s] [%(name)25s] %(message)s',
+        },
+        'instance': {
```
Comment (kinnou02) [inline tartare/default_settings.py]: ? :)

Comment (antoine-de) [inline tartare/default_settings.py]: gna gna gna

File: tartare/tasks.py [hunk 3]
```diff
@@ -1,15 +1,104 @@
...
+from shutil import copyfile
 import os
 
 @celery.task()
 def update_data_task():
     input_dir = app.config.get("INPUT_DIR")
     output_dir = app.config.get("OUTPUT_DIR")
-    move_data(input_dir, output_dir)
+    current_data_dir = app.config.get("CURRENT_DATA_DIR")
+    logging.info('scanning directory {}'.format(input_dir))
```
Comment (kinnou02) [inline tartare/tasks.py]: you should use
```python
logger = logging.getLogger(__name__)
logger.info('blabla %s', input_dir)
```

File: tartare/tasks.py [hunk 4]
```diff
@@ -1,15 +1,104 @@
...
+    handle_data(input_dir, output_dir, current_data_dir)
+
+def type_of_data(filename):
```
Comment (kinnou02) [inline tartare/tasks.py]: :cry:

File: tartare/default_settings.py (hunk 2 above)
Comment (pbougue) [inline tartare/default_settings.py]: ??

File: tartare/__init__.py (hunk 1 above)
Comment (datanel) [inline tartare/__init__.py]: we need app because of `app.logger.addHandler(handler)`

File: tartare/default_settings.py (hunk 2 above)
Comment (datanel) [inline tartare/default_settings.py]: @kinnou02 ???

Comment (kinnou02) [inline tartare/default_settings.py]: You don't use it

File: tartare/tasks.py (hunk 3 above)
Comment (datanel) [inline tartare/tasks.py]: done

File: tartare/default_settings.py (hunk 2 above)
Comment (datanel) [inline tartare/default_settings.py]: removed

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
OK for me

**Pull request merged**